android/src/main/java/com/swmansion/rnscreens/SearchViewFormatter.kt

```diff
@@ -0,0 +1,40 @@
+package com.swmansion.rnscreens
+
+import android.graphics.drawable.Drawable
+import android.view.View
+import android.widget.EditText
+import androidx.appcompat.widget.SearchView
+
+class SearchViewFormatter(var searchView: SearchView) {
+    private var mDefaultTextColor: Int? = null
+    private var mDefaultTintBackground: Drawable? = null
+
+    private val searchEditText
+        get() = searchView.findViewById<View>(androidx.appcompat.R.id.search_src_text) as? EditText
```

Comment:
We have a nullable type here because I used `as?`. I am not sure how would you like to inform about it being null. I don't think throwing an exception is a good way to go and warning in console can be easily missed.